qemu/tests/fwcfg.py

```diff
@@ -67,6 +67,9 @@ def run(test, params, env):
         if not windbg_installed:
             win_dump_utils.install_windbg(test, params, session,
                                           timeout=wdbg_timeout)
+        # A temporary workaround to clear up unexpected pop-up in guest
```

Comment:
# TODO: xxx

Reply:
Hi @vivianQizhu 
This is just a note to document these codes are a workaround for unexpected popups, I don't think we need to add the TODO keyword. Not sure if I understand your suggestion clearly.

Reply:
@leidwang If it is a temporary workaround, I assume we need a 'TODO' to indicates that we want to fix it later.

Reply:
Okay,updated the annotation.

Reply:
Hi @vivianQizhu Could you please review it again?Thanks!